specviz/plugins/statistics/statistics_widget.py

```diff
@@ -143,15 +190,17 @@ def _update_stat_widgets(self, stats):
             return
         for key in stats:
             if key in self.stat_widgets:
-                text = format_float_text(stats[key])
-                self.stat_widgets[key].setText(text)
+                print(key, " ", stats[key])
```

Comment:
Remove print statement.

Reply:
Sorry about this, I thought I had checkout a clean version of stuff, but clearly I hadn't 🤣